src/test/java/tsunami/security/scanner/utilities/AppTest.java

```diff
@@ -16,12 +16,115 @@
 
 package tsunami.security.scanner.utilities;
 
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.*;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.AppsV1Api;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1Service;
+import io.kubernetes.client.util.Yaml;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import org.junit.Rule;
 import org.junit.Test;
-import static org.junit.Assert.*;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
 
-public class AppTest {
-    @Test public void testAppHasAGreeting() {
-        App classUnderTest = new App();
-        assertNotNull("app should have a greeting", classUnderTest.getGreeting());
-    }
+@RunWith(JUnit4.class)
+public final class AppTest {
+
+  @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
+  @Rule public TemporaryFolder folder = new TemporaryFolder();
+
+  @Mock CoreV1Api mockCoreV1Api;
+  @Mock AppsV1Api mockAppsV1Api;
+
+  String resourceConfig =
```

Comment:
make this a constant: `private static final String RESOURCE_CONFIG = ...` and move the constant to the beginning of the class.